akka-actor/src/main/scala/akka/actor/LightArrayRevolverScheduler.scala

```diff
@@ -316,6 +317,7 @@ class LightArrayRevolverScheduler(config: Config, log: LoggingAdapter, threadFac
         }
         executeBucket()
         wheel(bucket) = putBack
+        spareTaskQueue = tasks
```

Comment:
In previous code we were creating a temporary `TaskQueue` to collect the nodes we wanted to put back. 

But here, we are adding all tasks back to the queue now. That same queue will be used again on next tick and will be carrying some baggage from the previous tick.

Reply:
`spareTaskQueue` is an empty queue. We save all nodes we want to put back to this queue and put the queue back to the wheel. After we process all nodes from `tasks`, this queue is empty now and we reuse it as a new `spareTaskQueue` for the next wheel bucket. And so on, on each step we reuse an empty queue from the previous step.

Reply:
Yes, you are perfectly right. Thanks.